core/app/models/workarea/pricing/calculators/tax_calculator.rb

```diff
@@ -50,6 +72,15 @@ def not_shipped_items_price_adjustments
         def price_adjustments_for(shipping)
           shipped_items_price_adjustments
         end
+
+        private
+
+        def user
+          return @user if defined? @user
+          return unless order.user_id.present?
+          @user = User.find(order.user_id)
+        rescue Mongoid::Errors::DocumentNotFound, Mongoid::Errors::InvalidFind
```

Comment:
This rescue won't cause `@user` to get memoized, so it'll run through the lookup again.